### ISSUE TITLE
SubMenu OnClick in not a function on onSubMenuClick

### ISSUE THREAD

**Issue by ditiem** [ant-design/ant-design#12751]
- [x ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.10.1

### Environment
browser, chrome

### Reproduction link
[https://codepen.io/ditiem/pen/bmjpde](https://codepen.io/ditiem/pen/bmjpde)

### Steps to reproduce
1. Go To https://ant.design/components/menu/
2. Make the browser window smaller so any of the horizontal menus will show and ellipsis (...) at the end
3. Open the code inspector
4. Click on any option of the "pop over" of the ellipsis of the menu

### What is expected?
Not to throw an error. It seems easy to fix, just ask if this.props.onClick is a function in the onSubMenuClick method (SubMenu.js:486)

### What is actually happening?
```Uncaught TypeError: e.props.onClick is not a function
    at Object.T.onSubMenuClick [as onClick] (index.js:1)
    at t.M.onClick (index.js:1)
    at onClick (index.js:1)
    at r.onClick (index.js:1)
    at Object.<anonymous> (index.js:1)
    at p (index.js:1)
    at index.js:1
    at x (index.js:1)
    at M (index.js:1)
    at T (index.js:1)
```

**Comment by pashist** [comment]
Faced same issue with 3.10.2

**Comment by ult** [comment]
same issue with 3.10.2

**Comment by picodoth** [comment]
taking a look

**Comment by picodoth** [comment]
a quick fix is made in https://github.com/react-component/menu/pull/216 and should be available in rc-menu@7.4.18

Please try updating dependencies and let us know if the issue is still there.

**Comment by ditiem** [comment]
@picodoth It works!

But in the version 3.12.3 the dependency of rc-menu is lower than 7.4.18 so the issue is still happening.

I edited by hand node_modules files to be sure it works :). Any chance antd will update this dependency soon?

Thanks a lot!!
